### PR TITLE
add printercolumn for syncStatus in SriovNetworkNodeState

### PR DESCRIPTION
**Pull request by tariq1890**
It would be useful to see the status of all nodes when querying through `SriovNetworkNodeStates`

### CODE REVIEW

**Review by SchSeba — APPROVED** [review]
I like the idea :)

**Review by adrianchiris — APPROVED** [review]

File: api/v1/sriovnetworknodestate_types.go [hunk 1]
```diff
@@ -90,6 +90,7 @@ type SriovNetworkNodeStateStatus struct {
 
 //+kubebuilder:object:root=true
 //+kubebuilder:subresource:status
+//+kubebuilder:printcolumn:name="Sync Status",type=string,JSONPath=`.status.syncStatus`
```
Comment (e0ne) [inline api/v1/sriovnetworknodestate_types.go:93]: Please, add something like `// +kubebuilder:printcolumn:name="Age",type=string,JSONPath=`.metadata.creationTimestamp``  (I didn't test this change with sriov operator so this line could be inacurate) to show resource age as we've got it now

**Review by e0ne — CHANGES_REQUESTED** [review]

File: api/v1/sriovnetworknodestate_types.go (hunk 1 above)
Comment (tariq1890) [reply api/v1/sriovnetworknodestate_types.go:93]: Sure!

**Comment by tariq1890** [comment]
@e0ne Added the printercolumn for Age. Please review

File: pkg/daemon/daemon.go [hunk 2]
```diff
@@ -103,14 +103,15 @@ type Daemon struct {
...
-	syncStatusFailed    = "Failed"
+	rdmaScriptsPath      = "/bindata/scripts/enable-rdma.sh"
+	udevScriptsPath      = "/bindata/scripts/load-udev.sh"
+	annoKey              = "sriovnetwork.openshift.io/state"
+	annoIdle             = "Idle"
+	annoDraining         = "Draining"
+	annoMcpPaused        = "Draining_MCP_Paused"
+	syncStatusSucceeded  = "Succeeded"
+	syncStatusFailed     = "Failed"
+	syncStatusInProgress = "InProgress"
```
Comment (e0ne) [inline pkg/daemon/daemon.go:114]: This change is not related to the PR but I'm OK with this minor cleanup

**Comment by tariq1890** [comment]
@e0ne Would you be able to review ?

**Comment by adrianchiris** [comment]
@e0ne can we merge this one ?

**Review by e0ne — APPROVED** [review]
LGTM. Thanks for addressing my comments

**Pull request merged**